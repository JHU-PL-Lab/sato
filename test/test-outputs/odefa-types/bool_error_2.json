[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "i1" ],
          "err_match_val": "input",
          "err_match_expected": "bool",
          "err_match_actual": "int"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [ "i2" ],
          "err_match_val": "input",
          "err_match_expected": "bool",
          "err_match_actual": "int"
        }
      ]
    ],
    "err_input_seq": [ 0, 0 ],
    "err_location": "ans = i1 and i2",
    "err_steps": 68
  }
]