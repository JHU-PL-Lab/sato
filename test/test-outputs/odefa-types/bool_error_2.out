Type errors for:
- Input sequence  : [0, 0]
- Found at clause : ans = i1 and i2
--------------------
* Value    : i1 = input
* Expected : bool
* Actual   : int
--------------------
* Value    : i2 = input
* Expected : bool
* Actual   : int
2 errors found. No further control flows exist.
